ext/nio4r/.clang-format

```diff
@@ -0,0 +1,14 @@
+---
+Language:        Cpp
```

Comment:
Is there some reason to have multiple spaces between `:` and `Cpp`?

Reply:
Nope, I'll fix that 😅...
(it was a copy an paste for clang's documentation)

Reply:
Haha, no problem. Thanks for fixing it.